### ISSUE TITLE
Can't init project

### ISSUE THREAD

**Issue by tipoc123** [expo/expo-cli#411]
## Description

I've installed expo. Then trying to create a new React Native project called "AwesomeProjectNew", but it doesn't create.

```
tipoc@tipoc-MS-7817:~/Desktop$ export EXPO_DEBUG=true
tipoc@tipoc-MS-7817:~/Desktop$ expo init AwesomeProjectNew
(0 , (_wordwrap || _load_wordwrap(...)).default)(...)(...).trimStart is not a function
TypeError: (0 , (_wordwrap || _load_wordwrap(...)).default)(...)(...).trimStart is not a function
    at /expo-cli@2.11.6/src/commands/init.js:105:19
    at Array.map (<anonymous>)
    at _callee$ (/expo-cli@2.11.6/src/commands/init.js:95:37)
    at tryCatch (/usr/lib/node_modules/expo-cli/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/usr/lib/node_modules/expo-cli/node_modules/regenerator-runtime/runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (/usr/lib/node_modules/expo-cli/node_modules/regenerator-runtime/runtime.js:114:21)
    at step (/usr/lib/node_modules/expo-cli/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /usr/lib/node_modules/expo-cli/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (/usr/lib/node_modules/expo-cli/node_modules/core-js/library/modules/_export.js:36:28)
```

## Environment

```
tipoc@tipoc-MS-7817:~/Desktop$ expo diagnostics

  Expo CLI 2.11.6 environment info:
    System:
      OS: Linux 4.15 Ubuntu 18.04.2 LTS (Bionic Beaver)
      Shell: 4.4.19 - /bin/bash
    Binaries:
      Node: 8.14.0 - /usr/bin/node
      npm: 6.4.1 - /usr/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    npmGlobalPackages:
      expo-cli: 2.11.6
```

**Comment by fson** [comment]
Looks like you need to update to node v10 or newer, at least until this code is fixed. Node 8 doesn't support `String.prototype.trimStart`.

**Comment by fson** [comment]
Fixed in 0fe61cb953009c91df02ee8a1ef4004cf5b08b0e. Thanks for reporting!
